packages/frontend/src/utils/getRpcUrl.ts

```diff
@@ -1,7 +1,35 @@
+import { L1_NETWORK } from 'src/constants'
 import { networks } from 'src/config'
+import { Chain } from './constants'
 
 export const getRpcUrl = (network: string) => {
   return networks[network]?.rpcUrl
 }
 
+export function getRpcUrlByNetworkName(networkName: string) {
```

Comment:
What is the purpose of this method if `getRpcUrl` already exists?

Reply:
haha! good point. i copy-pasted this block of code from `Transaction.tsx`